LabPortalAPI/Models/Program.cs

```diff
@@ -43,9 +43,11 @@
 {
     app.UseSwagger();
     app.UseSwaggerUI();
-    app.Run();
+    app.Run($"http://##.##.##.##:7282");
```

Comment:
I set up the program.cs so that you could launch the exe with parameters to specify the port. Please revert this change